packages/ionio/package.json

```diff
@@ -57,7 +57,7 @@
     "typescript": "^4.6.4"
   },
   "dependencies": {
-    "liquidjs-lib": "git+https://github.com/louisinger/liquidjs-lib.git#psbt-taproot-support"
+    "ldk": "git+https://github.com/louisinger/ldk.git#bip371-liquidjs"
```

Comment:
nACK

to avoid tsdx error, lets use ldk 0.5.1 AND liquidjs from git

Reply:
done